src/mmrelay/migrate.py

```
@@ -0,0 +1,1883 @@
+            "action": "move",
+            "dry_run": True,
+        }
+
+    # Proceed with migration
+    staging_dir = _get_staging_path(new_home, "gpxtracker")
+
+    try:
+        staging_dir.mkdir(parents=True, exist_ok=True)
+
+        migrated_count = 0
+        errors: list[str] = []
```

Ruff (0.14.14)

[error] Local variable `errors` is assigned to but never used

Remove assignment to unused variable `errors`

(F841)